alns/select/SelectionScheme.py

```diff
@@ -0,0 +1,101 @@
+from abc import ABC, abstractmethod
+from typing import Tuple, Optional
+
+import numpy as np
+from numpy.random import RandomState
+
+
+class SelectionScheme(ABC):
```

Comment:
Coverage is complaining about this class not being tested. But because we are using an abstract base class here, it doesn't seem [possible](https://stackoverflow.com/questions/9757299/python-testing-an-abstract-base-class) to test it. Any thoughts about this?